docs/_data/spacing.json

```diff
@@ -115,6 +115,10 @@
     {
       "prefix": "b",
       "values": "0, 1, 2, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 50, 100"
+    },
+    {
+      "prefix": "i",
+      "values": "0, 1, 2, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 50, 100"
```

Comment:
Does inset make sense with 50% and 100%?

Reply:
Nah, I don't think so. `inset: 50%` would be the same as `left: 50%; right: 50%; top: 50%; bottom: 50%;` which would only work well on an element with no dimensions. Same goes for `100%`.